### PR TITLE
Feat/bug fixes

### PR DESCRIPTION
**Pull request by anshul-pinto0410**
Updates: 
- Changed carousel to accommodate interaction
- removed sm height for inputAction
- minor cosmetic fixes

## Checklist

- [ ] PR is up-to-date with a description of changes and screenshots (if applicable).
- [ ] All files are lint-free.
- [ ] Followed Semantic PR guidelines.
- [ ] Tested locally for regressions.

### CODE REVIEW

File: src/components/Card/card.css [hunk 1]
```diff
@@ -47,3 +48,9 @@
     width: 40px;
     @apply lil-float-right lil-h-full;
 }
+
+@screen sm {
```
Comment (pritam248) [inline src/components/Card/card.css:52]: good for medium screens ?

File: src/components/Card/carouselcard.css [hunk 2]
```diff
@@ -23,71 +23,73 @@ The css colors have to be updated with tailwind utility colors
...
+    @apply lil-py-4 lil-btn-link lil-mr-2 lil-text-s-12 lil-font-bold;
+  }
 }
 
 .lil-carousel-card-footer-action:hover,
-.lil-carousel-card-footer-action:focus{
-    @apply lil-outline-none;
+.lil-carousel-card-footer-action:focus {
+  @apply lil-outline-none;
 }
```
Comment (pritam248) [inline src/components/Card/carouselcard.css]: move this above media query

File: src/components/Card/card.css (hunk 1 above)
Comment (anshul-pinto0410) [reply src/components/Card/card.css:52]: yep fixed h solved multiple sized cards issue.. checked for all sizes

Comment (anshul-pinto0410) [reply src/components/Card/card.css:52]: yep

**Review by pritam248 — APPROVED** [review]

**Pull request merged**